Updated changelog and docs

xrmx opened the pull request:
* Changelog uses new format of giving a link to the given PRs
* Supersedes #310, #300, #292

[xrmx: same as #344 but rebased]

Andrew-Chen-Wang commented:
@mrmx Thanks for doing this! The one thing I forgot: the other rule. I can only merge PRs that pass the CI. You would first have to create a PR from #349 before I can merge any new PRs.

Andrew-Chen-Wang commented:
Updated here.

xrmx commented:
@Andrew-Chen-Wang I'm updating the branch, adding the pyjwt pr fix to the changelog.

Andrew-Chen-Wang commented:
@mrmx Ah apologies. Master branch merged into this already. Is there any other changes that you're trying to add?

xrmx commented:
I've added the following line to latest:

```
* Fixed support for PyJWT>=2.0.0 ([#376](https://github.com/SimpleJWT/django-rest-framework-simplejwt/pull/376))
```

xrmx commented:
@Andrew-Chen-Wang If you merge this I can rebase the last one :)
